service/src/test/java/uk/gov/hmcts/reform/fpl/service/PastHearingDatesValidatorServiceTest.java

```diff
@@ -43,4 +47,24 @@ void shouldReturnValidationErrorsWhenHearingDateTimeIsNotValid() {
 
         assertThat(validationErrors).containsExactlyInAnyOrder("Enter a valid start time", "Enter a valid end time");
     }
+
+    @ParameterizedTest(name = "{0}")
+    @MethodSource("validateHearingDatesSource")
+    void shouldReturnValidationErrorWhenHearingEndDateIsNotAfterTheStartDate(
+        String testName, LocalDateTime hearingStartDate, LocalDateTime hearingEndDate
+    ) {
+        final List<String> validationErrors = service.validateHearingDates(hearingStartDate, hearingEndDate);
+
+        assertThat(validationErrors)
+            .containsOnly("The end date and time must be after the start date and time");
+    }
+
+    private static Stream<Arguments> validateHearingDatesSource() {
+        LocalDateTime startDateTime = LocalDateTime.of(2020, 12, 10, 10, 10, 0);
+        return Stream.of(
```

Comment:
This looks ok, but could use the closest boundary possible i.e. Testing minus/plus the minimum recoded (minutes/seconds)?

Reply:
updated test data with seconds/minutes difference